DOCS: Update documentation for MonoBehaviour OnMouse events (ISX-2383) (#2301)
Unity-Technologies/InputSystem: Packages/com.unity.inputsystem/Documentation~/Migration.md

### Mouse

> The MonoBehaviour.OnMouse events such as [MonoBehaviour.OnMouseDown()](https://docs.unity3d.com/ScriptReference/MonoBehaviour.OnMouseDown.html) are supported using the Unity Editor 6.4 or newer. 

Reviewer: Slight, very subjective, rewording suggestion:

The MonoBehaviour.OnMouse events (for example, [MonoBehaviour.OnMouseDown()](https://docs.unity3d.com/ScriptReference/MonoBehaviour.OnMouseDown.html)) are only supported in Unity Editor 6.4 and later.

Or

The MonoBehaviour.OnMouse events (for example, [MonoBehaviour.OnMouseDown()](https://docs.unity3d.com/ScriptReference/MonoBehaviour.OnMouseDown.html)) are supported in Unity Editor 6.4 and later.

Also, not sure if calling them methods or events is better here

> The MonoBehaviour.OnMouse events such as [MonoBehaviour.OnMouseDown()](https://docs.unity3d.com/ScriptReference/MonoBehaviour.OnMouseDown.html) are supported using the Unity Editor 6.4 or newer. 

Reviewer 2: ```suggestion
`MonoBehaviour.OnMouse` events, such as [MonoBehaviour.OnMouseDown](https://docs.unity3d.com/ScriptReference/MonoBehaviour.OnMouseDown.html), are supported in Unity 6.4 and later.  
```
